crates/sui-e2e-tests/tests/per_epoch_config_stress_tests.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_core_types::language_storage::TypeTag;
+use rand::random;
+use std::path::PathBuf;
+use std::sync::Arc;
+use sui_json_rpc_types::SuiTransactionBlockEffectsAPI;
+use sui_macros::sim_test;
+use sui_types::base_types::{EpochId, ObjectID, ObjectRef, SuiAddress};
+use sui_types::transaction::{CallArg, ObjectArg};
+use sui_types::{SUI_DENY_LIST_OBJECT_ID, SUI_FRAMEWORK_PACKAGE_ID};
+use test_cluster::{TestCluster, TestClusterBuilder};
+
+const DENY_ADDRESS: SuiAddress = SuiAddress::ZERO;
+
+#[ignore]
```

Comment:
why ignored?

Reply:
Because the Move APIs are not public callable yet.